config/routes.rb

```diff
@@ -1,9 +1,20 @@
 Rails.application.routes.draw do
 
   root "users#index"
-  resources :users, only: %i[show index]
-  get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
-  get '/users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
+  resources :users, only: [:show] do
+    resources :posts, only: [:index, :show]
+  end
+
+  resources :posts, only: [:new, :create] do
+    resources :comments, only: [:new, :create]
```

Comment:
- [ ] Great job with your implementation of the comments. However, per the project requirements, not only should be a pair of routes in the `config/routes.rb` file for creating comments but you should also ensure that there is a separate form in the `views/comments/new.html.erb` file that accepts input for creating comments. :+1:  So kindly implement this as well :+1: